nilearn/decomposition/multi_pca.py

```diff
@@ -2,19 +2,21 @@
 PCA dimension reduction on multiple subjects
```

Comment:
I think it would be worth adding a few more explanations on what this means in practice.